quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildAndStartDistTest.java

```diff
@@ -41,7 +41,7 @@ public class BuildAndStartDistTest {
     void testBuildAndStart(KeycloakDistribution dist) {
         RawKeycloakDistribution rawDist = dist.unwrap(RawKeycloakDistribution.class);
         // start using based on the build options set via CLI
-        CLIResult cliResult = rawDist.run("build", "--storage=chm");
+        CLIResult cliResult = rawDist.run("build", "--http-relative-path=/auth");
```

Comment:
I believe the changes for the `http-relative-path` option are unnecessary in this test.

Reply:
thank you, I have removed the `http-relative-path` from this line, but other occurrences in this test seems to necessary, please tell me if you think otherwise.

Reply:
I'm just trying to find the correlation between the `storage` and `http-relative-path` options, as they should not be dependent at all. I also tried to manually follow steps in the test without the specified option `http-relative-path` in `keycloak.conf`, and everything works as expected.

@vramik Could you please verify your PR does not touch anything else regarding this functionality? Thanks

Reply:
@mabartos - in some places the tests (at least looking at the test's name) use just some build option to test some behavior. So for those tests that still want to have just any build option, `--storage` can be IMHO replaced with another build option in those cases.

Reply:
@ahus1 Ahh, yep, you're right. I've been just curious whether the functionality changed somehow or not. In this case, I'm ok with that if @vramik confirms the intention behind it - stated as 'seems to be necessary'.

Reply:
That's correct, I've used the `http-relative-path` as a random replacement for `storage` which is being removed. I'm sorry I misunderstood your comment, I thought you were suggesting that the test should work as expected even without using the build option, that's why I wrote that "it seems necessary".

Reply:
@vramik Thanks for the confirmation!